### ISSUE TITLE
Thanks!  You've saved me many hours of time!

### ISSUE THREAD

**Issue by coddingtonbear** [wez/govee-lan-hass#1]
I literally set aside my evening tonight to write essentially the same library you created here as [`govee-py`](https://github.com/wez/govee-py), and was thrilled to see that you'd already done it, and not only that -- you'd done it in a much more thoughtful way than I was originally roughly planning to.  This appears to work great for all four of my govee lights and has none of the unreliability of the normal hacs govee integration caused by govee's servers' rate limits.  I'm sure we'll see more folks moving to this hacs integration you've created here as more govee devices become enabled for LAN control.

As an aside, and this might very well be a problem in just my environment, but I did have to install the `govee-led-wez` dependency manually for some reason.  I'm assuming it's just a problem in my environment, though, because it does look like you've declared it correctly in your manifest.  That said, I thought I should say something in case somebody else encounters the same trouble.

Cheers & thanks again!

**Comment by wez** [comment]
Thanks for the kind words!

> As an aside, and this might very well be a problem in just my environment, but I did have to install the `govee-led-wez` dependency manually for some reason. I'm assuming it's just a problem in my environment, though, because it does look like you've declared it correctly in your manifest

There were some issues with it a few weeks ago, but it definitely works now: I've tested this with several fresh recreations of my hass docker image and it does successfully install.

It's possible that the minimum hass version needs to be updated in the manifest: there's some bluetooth related stuff that might require 2022.12.something.

**Comment by VynDesign** [comment]
Seconded! This was amazingly well-timed for me, as I just got two sets of the H7060 outdoor flood lights. I had used the cloud-based Govee integration and woke up today to find myself rate-limited. I went searching for a postman collection or tutorial on how to connect using the LAN API and found this integration! Installed, discovered, and automated within minutes.

I did have some system instability after installing, but a hard reboot seems to have taken care of that (and I'm not certain it had anything to do with this integration, but the timing is a bit suspect).
